tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/RequestValidationHelperTests.cs

```diff
@@ -73,7 +73,7 @@ public void ClientAssertionRequestValidatorMismatchParameterTest()
             credential.Audience = Audience2;
 
             // cached assertion should be invalid
-            Assert.IsFalse(ClientCredentialHelper.ValidateClientAssertion(credential, new AuthorityEndpoints(null, null, Audience2), false));
+            Assert.IsFalse(ClientCredentialHelper.ValidateClientAssertion(credential, new AuthorityEndpoints(null, null, Audience1), false));
```

Comment:
not Audience2?

Reply:
No, I debugged this some more.  it does the checks against audience1 to make sure that things fail when the cred changes to audience2.